subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
+            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Tutorials</a></li>
         </ul>
 
         <h3 id="reference">Reference</h3>
         <ul>
-            <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL Reference</a></li>
-            <li><a href="/dsl/">Groovy DSL Reference</a></li>
-            <li><a href="/javadoc/">Gradle API Javadoc</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#gradle-api" aria-expanded="false" aria-controls="gradle-api">Gradle API</a>
+                <ul id="gradle-api">
+                    <li><a href="/javadoc/">Javadoc</a></li>
+                    <li><a href="/dsl/">Groovy DSL</a></li>
+                    <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL</a></li>
+                </ul>
+            </li>
             <li><a href="/userguide/command_line_interface.html">Command-Line Interface</a></li>
             <li><a href="/userguide/plugin_reference.html">Core Plugins</a></li>
+            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
             <li><a href="https://docs.gradle.com/build-scan-plugin">Build Scan Plugin User Manual</a></li>
         </ul>
 
-        <h3 id="getting-started">Managing Gradle Projects</h3>
+        <h3 id="getting-started">Running Gradle Builds</h3>
         <ul>
-            <li><a href="https://guides.gradle.org/creating-new-gradle-builds/">Creating New Gradle Builds</a></li>
-            <li><a href="https://guides.gradle.org/creating-build-scans/">Inspecting Gradle Builds</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#organizing-gradle-projects" aria-expanded="false" aria-controls="organizing-gradle-projects">Organizing Project Structure</a>
-                <ul id="organizing-gradle-projects">
-                    <li><a href="/userguide/organizing_gradle_projects.html">Organizing Projects Best Practices</a></li>
-                    <li><a href="/userguide/directory_layout.html">Directory Layout</a></li>
-                    <li><a href="/userguide/composite_builds.html">Integrating Separate Gradle Builds (Composite Builds)</a></li>
-                    <li><a href="/userguide/intro_multi_project_builds.html">Multi-Project Builds</a></li>
+            <li>Using the Command Line (TBD)</li>
+            <li>Using Gradle From an IDE (TBD)</li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#customizing-execution" aria-expanded="false" aria-controls="customizing-execution">Customizing Execution</a>
+                <ul id="customizing-execution">
+                    <li><a href="/userguide/build_environment.html">Configuring the Build Environment</a></li>
+                    <li><a href="/userguide/gradle_daemon.html">Configuring the Gradle Daemon</a></li>
```

Comment:
This seems suited to the Appendix as the wrapper, though honestly isn't in good shape.

Reply:
Don't understand what the comment is trying to say, but it is similar to the Wrapper chapter in style and probably should go into the appendix (or wherever we end up putting these items).